### ISSUE TITLE
MSDK dispatcher can't load libmfx64-gen.dll on Windows

### ISSUE THREAD

**Issue by nyanmisaka** [Intel-Media-SDK/MediaSDK#2904]
## System information
- CPU information (link to ark.intel.com): i7-1165G7 Tiger Lake
- GPU information (link to ark.intel.com): Xe 96EU
- Driver version: 30.0.101.1340

### Describe the current behavior
- MSDK dispatcher can't load `libmfx64-gen.dll` on Windows

### Describe the expected behavior
- The Windows version of MSDK dispatcher should have forward compatibility with new VPL runtime like the behavior on Linux.

Related code are missing in MSDK dispatcher:
https://github.com/Intel-Media-SDK/MediaSDK/blob/0bf5ca1f9519ce1e6ee2674ba777680786310510/api/mfx_dispatch/windows/src/mfx_load_dll.cpp#L30

But are only available in VPL dispatcher:
> const wchar_t *const defaultOneVPLDLLName   = { L"libmfx64-gen.dll" };
https://github.com/oneapi-src/oneVPL/blob/544a5acd36ca259f3df25321be500f160024a054/dispatcher/windows/mfx_load_dll.cpp#L17

**Comment by dmitryermilov** [comment]
Hi @nyanmisaka ,

Do you have a real reproducer for the issue?

On Windows part of loading logic is hidden inside C:\Windows\System32\libmfxhw64.dll. It's s kind of dynamic dispatcher.

**Comment by nyanmisaka** [comment]
Thanks! that make sense to me. My driver was downgraded by Windows to an old version that doesn't support VPL. I revert these changes then it works. Feel free to close this issue!

BTW Is there any environment variable to switch between MSDK and VPL like on Linux?
```
export INTEL_MEDIA_RUNTIME=ONEVPL  # for VPL Runtime: libmfx-gen.so.1.2
or
export INTEL_MEDIA_RUNTIME=MSDK    # for Media SDK Runtime: libmfxhw64.so.1
```

**Comment by dmitryermilov** [comment]
NP!

> BTW Is there any environment variable to switch between MSDK and VPL like on Linux?

You wanted to say "on Windows"?

**Comment by nyanmisaka** [comment]
yes on Windows. it’s a typo

**Comment by dmitryermilov** [comment]
There's no such option. We provided INTEL_MEDIA_RUNTIME to **workaround** the issue with dual GPUs. On Windows a regular way to select GPU(and underlying run-time: MSDK or VPL) is to use MFX_IMPL_HARDWARE/MFX_IMPL_HARDWARE2/MFX_IMPL_HARDWARE3. But we never enabled this on Linux and always demanded app to set GPU handle via SetHandle() call. But SetHandle() doesn't help to load "right" library instance (because SetHandle is applied to already loaded run-time:) ). 
As a workaround we enabled INTEL_MEDIA_RUNTIME. Pleas don't misuse INTEL_MEDIA_RUNTIME, it's only for "On the multi GPU system which has both "Legacy" and "VPL" GPUs".
